src/huggingface_hub/utils/_http.py

```diff
@@ -48,6 +49,68 @@
 
 logger = logging.get_logger(__name__)
 
+
+@dataclass(frozen=True)
+class RateLimitInfo:
```

Comment:
let's expose `RateLimitInfo` and `parse_ratelimit_headers` in `huggingface_hub.utils` ?